include/dungeon_message.h

```diff
@@ -0,0 +1,54 @@
+#ifndef GUARD_DUNGEON_MESSAGE_H
+#define GUARD_DUNGEON_MESSAGE_H
+
+#include "structs/dungeon_entity.h"
+#include "code_80130A8.h"
+
+struct MonDialogueSpriteInfo
+{
+    s16 species;
+    u8 spriteId;
+};
+
+struct DungeonDialogueStruct
+{
+    u16 unk0;
+    u8 unk2;
+    u8 unk3;
+    s16 unk4;
+    s16 unk6;
+    const u8 *str;
+};
+
+void sub_80521D0(void);
+void sub_8052210(bool8 a0);
+
+// These functions display the string if certain conditions are met. The conditions differ depending on the function. The string is then saved to the message log.
+void TryDisplayDungeonLoggableMessage(Entity *pokemon, const char *str);
+void TryDisplayDungeonLoggableMessage2(Entity *r0, const char *str);
```

Comment:
Should probably keep the same standard of `r0` -> `pokemon` and `r1` -> `target` for these funcs like all the rest.

Reply:
Good idea